packages/server/src/controllers/index.controller.ts

```diff
@@ -19,21 +22,40 @@ export class IndexController extends Controller {
   ): Promise<IStandardResponse<string>> {
     try {
       const webhookAuth = validateWebhook(livepeerSignature, payload);
+      console.log(payload);
 
       if (!webhookAuth) {
         console.log('Invalid signature or timestamp');
         return SendApiResponse('Invalid signature or timestamp', null, '401');
       }
       switch (payload.event) {
-        case 'asset.ready':
-          await startAITools(payload.payload.id);
+        case LivepeerEvent.assetReady:
```

Comment:
@Eric-Vondee can you take a look at this please

Reply:
Good. You can move the code for assetReady to a private function.